server/odc-service/src/main/java/com/oceanbase/odc/service/session/OdcStatementCallBack.java

```diff
@@ -176,6 +176,9 @@ public List<JdbcGeneralResult> doInStatement(Statement statement) throws SQLExce
                 if (thrown.get() == null || !stopWhenError) {
                     try {
                         executeResults = doExecuteSql(statement, sqlTuple);
+                        if (executeResults.size() == 1 && executeResults.get(0).getThrown() != null) {
```

Comment:
there is a filed called `status` in `JdbcGeneralResult` that can be used to judge if the execution is succeed

Reply:
this field is not usable. The `status` will only be failed when the code enters line 192.

Reply:
done , fixed this.